README.md

```diff
@@ -62,7 +62,7 @@ To configure your project manually, follow these steps:
    ```json
    "main": "lib/commonjs/index.js",
    "module": "lib/module/index.js",
-   "react-native": "src/index.js",
+   "react-native": "lib/module/index.js",
```

Comment:
Thanks for the PR. This should be `index.tsx` for TypeScript projects (unless you're supporting old versions of React Native without TypeScript support):

```suggestion
   "react-native": "src/index.tsx",
```